conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Tables.scala

```diff
@@ -1736,8 +1736,8 @@ trait Tables {
     /** Index over (level) (database name endorsing_rights_level_idx) */
     val index2 = index("endorsing_rights_level_idx", level)
 
-    /** Index over (delegate) (database name ix_delegate_slot) */
-    val index3 = index("ix_delegate_slot", delegate)
+    /** Index over (delegate,slot) (database name ix_delegate_slot) */
+    val index3 = index("ix_delegate_slot", (delegate, slot))
```

Comment:
is this related to this PR?

Reply:
I regenerated the files with the new generator and this misalignment between definition and sources came out, so yet and no.
But I don't think it's worth its own fix PR